sdk/keyvault/keyvault-secrets/package.json

```
@@ -19,51 +19,44 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "main": "./dist/index.js",
-  "module": "dist-esm/src/index.js",
-  "types": "./types/keyvault-secrets.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "engines": {
     "node": ">=18.0.0"
   },
   "files": [
-    "types/keyvault-secrets.d.ts",
     "dist/",
-    "dist-esm/src",
     "README.md",
     "LICENSE"
   ],
-  "browser": {
-    "os": false,
-    "process": false
-  },
+  "browser": "./dist/browser/index.js",
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build": "npm run clean && tsc -p . && npm run build:nodebrowser && dev-tool run extract-api",
-    "build:browser": "tsc -p . && dev-tool run bundle",
-    "build:node": "tsc -p . && dev-tool run bundle",
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
+    "build:browser": "echo skipped",
+    "build:node": "dev-tool run build-package && dev-tool run bundle",
```

Is bundling still needed? Looks like browser tests are skipped.
```suggestion
    "build:node": "dev-tool run build-package",
```